en/tutorials/tuning_the_ecl_ekf.md

```diff
@@ -323,15 +327,17 @@ Poor GPS accuracy is normally accompanied by a rise in the reported velocity err
 
 ![](../../assets/ecl/gps_glitch_-_test_ratios.png)
 
-This is accompanied with rise in the GPS receivers reported velocity accuracy which indicates that it was likely a GPS error.  
+This is accompanied with rise in the GPS receivers reported velocity accuracy which indicates that it was likely a GPS error.
+
 ![](../../assets/ecl/gps_glitch_-_reported_receiver_accuracy.png)
 
-If we also look at the GPS horizontal velocity innovations and innovation variances, we can see the large spike in North velocity innovation that accompanies this GPS 'glitch' event.  
+If we also look at the GPS horizontal velocity innovations and innovation variances, we can see the large spike in North velocity innovation that accompanies this GPS 'glitch' event.
+
 ![](../../assets/ecl/gps_glitch_-_velocity_innovations.png)
 
 ### Determination of GPS Data Loss
 
-Loss of GPS data will be shown by the velocity and position innvoation test ratios 'flat-lining'. If this occurs, check the oher GPS status data in vehicle\_gps\_position for further information.
+Loss of GPS data will be shown by the velocity and position innovation test ratios 'flat-lining'. If this occurs, check the other GPS status data in `vehicle_gps_position` for further information.
 
 \(insert example plots showing loss of GPS data here\)
```

Comment:
These images are missing

Reply:
I haven't yet received logs where GPS data was lost which is different to a temporary glitch. I will generate data using SITL.